[4.0 CTP 2] crash when running `npx wolkenkit dev` on windows

### What is this bug about?

Following the readme it says to run `npx wolkenkit dev` but a crash happens when doing so.

### What is the expected result?

It should start development

### What is the actual result?

```
❯ npx wolkenkit dev
\ error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/domain/communication/message.js': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/domain/communication/message.js.map': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/views/messages/MessagesItem.js': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/views/messages/MessagesItem.js.map': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/views/messages/index.js': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/views/messages/index.js.map': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/views/messages/initializer.js': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/views/messages/initializer.js.map': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/views/messages/projections/liked.js': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/views/messages/projections/liked.js.map': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/views/messages/projections/sent.js': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/views/messages/projections/sent.js.map': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/views/messages/queries/all.js': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/server/views/messages/queries/all.js.map': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/test/domain/communication/messageTests.js': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
error TS5033: Could not write file 'C:/dev/me/experiments/wolkenkit-experiment/'C:/dev/me/experiments/wolkenkit-experiment/build'/test/domain/communication/messageTests.js.map': EINVAL: invalid argument, mkdir 'C:/dev/me/experiments/wolkenkit-experiment/'C:'.
```
### What steps are needed to reproduce the bug?

Run `npx wolkenkit dev`

### What else should we know?

Im on windows 10 running node 14.4.0

--- a/lib/common/application/compileWithTypeScript.ts
+++ b/lib/common/application/compileWithTypeScript.ts
@@ -14,11 +14,13 @@ const compileWithTypeScript = async function ({
     throw new errors.CompilationFailed('Source folder does not exist.');
   }
 
+  const shellQuote = process.platform === 'win32' ? `"` : `'`;
+
   const { code, stdout, stderr } = shell.exec(oneLine`
     npx tsc
       --module CommonJS
       --noEmitOnError
-      --outDir '${targetDirectory}'
+      --outDir ${shellQuote}${targetDirectory}${shellQuote}
   `, { cwd: sourceDirectory });
 
   if (code !== 0) {

